girder_annotation/girder_large_image_annotation/models/annotation.py

```diff
@@ -437,6 +437,63 @@ class AnnotationSchema:
         'description': 'An image overlay on top of the base resource.',
     })
 
+    segmentsToColorSchema = {
+        'type': 'array',
+        'items': colorSchema,
+        'description': 'An array where the indices correspond to pixel '
+                       'values in the pixel map image, and the values '
+                       'correspond to colors.'
+    }
+
+    categoryColorMap = {
+        'type': 'object',
+        'properties': {
+            'segments': {
```

Comment:
I think this would be better as 
```
'values': {
    'type': 'array',
    'items': {'type': 'integer'},
}
```
Using `values` for consistency with other annotations, and storing just indices into the category map rather than strings.  Since we could have millions of pixels, using integers here will be more efficient.

The `colormap` might conceptually be better called `categories` where each entry has a color, a label string, and an optional description to give a more verbose description of the category.

For pixelmaps with boundaries, perhaps this value array should be half as long as the actual number of number of distinct indices and then the category would have a `fillColor` and a `strokeColor` (and for non-boundaries pixelmaps, this would just use the `fillColor`).

Reply:
I've implemented these changes in 47c74f4.

I've also removed having two representations of the `values` array. Let me know if there are any further refinements to be made.